scripts/src/utils/resolvePaths.ts

```diff
@@ -1,8 +1,6 @@
-'use strict';
-
+import findUp from 'find-up';
 import fs from 'fs';
 import path from 'path';
-import findUp from 'find-up';
 
 export function nodeModulesToRoot(): string[] {
```

Comment:
Unnecessary empty function?

Reply:
It shouldn't be?

https://github.com/microsoft/fluentui-react-native/blob/3b2161653406028ea64b4a3949454e940cf09ba6/scripts/src/utils/resolvePaths.ts#L5-L17

Reply:
Ah. Misread sorry